algorithms/src/std_algorithms/Kokkos_ExclusiveScan.hpp

```diff
@@ -23,38 +23,46 @@
 namespace Kokkos {
 namespace Experimental {
 
+//
+// overload set accepting execution space
+//
+
 // overload set 1
 template <class ExecutionSpace, class InputIteratorType,
           class OutputIteratorType, class ValueType>
 std::enable_if_t<::Kokkos::Experimental::Impl::are_iterators<
-                     InputIteratorType, OutputIteratorType>::value,
+                     InputIteratorType, OutputIteratorType>::value&& ::Kokkos::
+                     is_execution_space<ExecutionSpace>::value,
```

Comment:
That is weird formatting...

Does it get any better if you use the `_v` versions instead of `::value` versions?

(Anyway, this is minor.)

Reply:
i updated it and seems to me it does not improve